File: app/src/main/java/moe/notify/animenotifier/domain/model/animelist/WatchingAnime.java
package moe.notify.animenotifier.domain.model.animelist;

import moe.notify.animenotifier.domain.model.anime.AnimeTitle;

/**
 * Created by GOKU on 24/08/2016.
 */

public class WatchingAnime {


    public AnimeProvider animeProvider;
    public AiringDate airingDate;
    public String preferredTitle;
    public AnimeTitle title;
    public String image;
    public Long id;
    public Episodes episodes;
}

File: app/src/main/java/moe/notify/animenotifier/domain/model/animelist/WatchingAnime.kt
package moe.notify.animenotifier.domain.model.animelist

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import moe.notify.animenotifier.domain.model.anime.AnimeTitle


open class WatchingAnime(
    open var animeProvider: AnimeProvider? = null,
    open var airingDate: AiringDate? = null,
    open var preferredTitle: String? = null,
    open var title: AnimeTitle? = null,
    open var image: String? = null,
    @PrimaryKey
    open var id: Long? = null,
    open var episodes: Episodes? = null) : RealmObject()